### ISSUE TITLE
Manually closed crowdsale endtime not returned

### ISSUE THREAD

**Issue by achamely** [mastercoin-MSC/mastercore#114]
The closedearly and endtime values for a crowdsale that is manually closed are not being updated/returned by mastercore:  ex for maidsafe below

ubuntu@ip-172-31-36-124:~/omniEngine$ python msc-cli.py getcrowdsale_MP 3
{
  "error": null, 
  "id": null, 
  "result": {
    "active": false, 
    "amountraised": "93249.21601551", 
    "closedearly": "unknown", 
    "deadline": 1400749200, 
    "earlybonus": 10, 
    "endedtime": "unknown", 
    "issuer": "1ARjWDkZ7kT9fwjPrjcQyvbXDkEySzKHwu", 
    "name": "MaidSafeCoin", 
    "percenttoissuer": 0, 
    "propertyiddesired": 1, 
    "starttime": 1398154020, 
    "tokensissued": "452552412", 
    "tokensperunit": "3400"
  }
}

**Comment by ghost** [comment]
Confirmed bug exists on tip

**Comment by zathras-crypto** [comment]
Not a bug, functionality is not implemented.  Persistence does not store
the time a crowdsale was closed nor the reason.

I raised this a number of times back in the day, and fought my corner that
we needed to provide the closing time and reason a crowdsale was closed.

I lost the debate tho, view was that dead.log contains the necessary info
and not important enough to prioritize persisting it/making available on
RPC layer.

I still stand by my original position that product is incomplete without it.

Thanks
Z

Confirmed bug exists on tip
